docs/changelog.md

```diff
@@ -31,6 +31,49 @@ permalink: /changelog/
 * **feature-prompts**, **browser-engine-gecko***
   * Added support for [Pop-up windows dialog](https://support.mozilla.org/en-US/kb/pop-blocker-settings-exceptions-troubleshooting#w_what-are-pop-ups).
 
+* **service-firefox-accounts**
+  * Added `FxaAccountManager`, which encapsulates a lower level accounts API and provides an observable interface for consumers that wish to be notified of account and profile changes.
+  * Background-worker friendly.
+  ```kotlin
+  // Long-lived instance, pinned on an application.
+  val accountManager = FxaAccountManager(context, Config.release(CLIENT_ID, REDIRECT_URL), arrayOf("profile"))
+  launch { accountManager.init() }
```

Comment:
would consumers ever need to handle exceptions here?

Reply:
They should be emitted via onError; internally the state machine will end up in a serviceable state one way or another. This could encounter a wide set of exceptions - failures to read/parse a persisted account, failures coming out of fxaclient, etc. I’d like the FxaAccountManager to express the error conditions more explicitly, but it’s not yet clear to me which ones are worth capturing. Plan is to let these emerge naturally from how we’ll end up using it in the consuming code, and remove pain points over time by reifying states.